chat-plugins/info.js

```diff
@@ -1407,6 +1429,18 @@ exports.commands = {
 		return this.sendReply("|raw|" + buf.join("") + "");
 	},
 
+	'!viewbanlist': true,
+	viewruleset: 'viewbanlist',
+	ruleset: 'viewbanlist',
+	banlist: 'viewbanlist',
+	viewbanlist: function (target, room, user, connection, cmd) {
+		if (!target) {
+			return this.sendReply("Usage: " + cmd + " <format>");
```

Comment:
This will say:
``Usage: viewbanlist <format>``

I think you forgot the ``/`` in the UI. Also, new code should probably be using template strings.